Documentation/DataFormats/T3datastructure/Index.rst

```diff
@@ -9,19 +9,30 @@ T3DataStructure
 
 TYPO3 offers an XML format, T3DataStructure, which defines a
 hierarchical data structure. In itself the data structure definition
-doesn't do much - it is only a back bone for higher level applications
+does not do much - it is only a back bone for higher level applications
 which can add their own configuration inside.
 
-Such applications can be:
-
-- :ref:`FlexForms <t3tca:columns-flex>` - a TCEform type which will allow users to build
-  information hierarchies (in XML) according to the Data Structure. In
-  this sense the Data Structure is like a DTD (Document Type
-  Definition) for the backend which can render a dynamic form based on
-  the Data Structure;
-
-- :t3ext:`templavoila` -
-  an extension which uses the Data Structure as backbone for mapping template HTML to data.
+The T3DataStructure could be used for different applications in theory, however it
+is commonly only used in the context of Flexforms.
+
+FlexForms are used in the contexts:
+
+-  TCA form type :ref:`FlexForms <t3tca:columns-flex>`:
+   The type allows users to build
+   information hierarchies (in XML) according to the data structure. In
+   this sense the Data Structure is like a DTD (Document Type
+   Definition) for the backend which can render a dynamic forms based on
+   the Data Structure.
+   
+-  The configuration of plugins of many common extensions with flexforms like 
```

Comment:
The term should be either flexforms or FlexForms but spelling should be consistent.

Reply:
Bulk changed them to FlexForms